LibSWBF2/Wrappers/Level.cpp

```diff
@@ -165,6 +180,17 @@ namespace LibSWBF2::Wrappers
 			World world;
 			if (World::FromChunk(p_MainContainer, worldChunk, world))
 			{
+				FNVHash worldName = FNV::Hash(world.GetName());
+				for (int i = 0; i < m_Lights.Size(); i++)
+				{
+					auto& light = m_Lights[i];
+
+					if (light.m_WorldName == worldName)
+					{
+						world.m_Lights.Add(light);
+					}
+				}
+
```

Comment:
As mentioned in the notes, the next PR has a proposal for a ```Config``` wrapper that handles all chunks (there are at least 7) which conform to the ```DATA``` - ```SCOP``` pattern ```lght``` does.  Worlds require specific lights, thus some way of attached them is needed.  Ultimately, worlds refer to lighting config chunks by their hashed name, which is demonstrated in the next PR.  This is a temporary solution to avoid breaking the public API.